guides/common/modules/snip_guide-not-ready.adoc

```diff
@@ -1,4 +1,4 @@
-This guide is not ready yet.
+== This guide is not ready yet
```

Comment:
I am unsure about this change. Is it necessary for asciidoctor? If so, we should rename the file because it now has a heading and therefore is no longer a snippet.

Reply:
Yes, for some reason, the build-html check started failing when I added the ribbons. I think you're right about this not being a snippet anymore.

Reply:
https://github.com/theforeman/foreman-documentation/pull/3661